src/app/component/header/header.component.html

```diff
@@ -97,7 +90,10 @@ <h1>Jyoti Technosoft LLP</h1>
             </div>
             <div class="credits">
               Designed by
-              <a href="https://jyotitechnosoft.com/" target="_blank"
+              <a
+                href="https://jyotitechnosoft.com/"
+                rel="noopener noreferrer"
+                target="_blank"
```

Comment:
on line 119 Please change this css style by adding align-items-center to class 
  style="align-items: center"
>

Reply:
On line 124 Please change inline css to external css 
Same on 143 ,152

Reply:
yes I have Remove popup box about and Alert button